build.rs

```diff
@@ -44,6 +44,7 @@ fn main() {
     let mut builder = bindgen::Builder::default()
         .header("ffi/webgpu-headers/webgpu.h")
         .header("ffi/wgpu.h")
```

Comment:
Don't need to specify it here if including via include path.

```suggestion
    let mut builder = bindgen::Builder::default()
        .header("ffi/wgpu.h")
```

Reply:
Out at dinner at the moment, will try this in 90 minutes or so.

Reply:
Okay, removed the include of `ffi/webgpu-headers/webgpu.h` and confirmed that the generated `bindings.rs` remains the same.